### DISCUSSION TITLE
Option to force constant encoding framerate

### DISCUSSION THREAD

**Discussion by danielmartina** [LizardByte/community#301] [Sunshine]
Hi,

I am using Sunshine with the Android Moonlight Client, which handles the oscillations in incoming framerate pretty bad, and produces stutters when moving the cursor after static screen on desktop apps like a browser.

My question and feature request is whether there is an option to force a constant frame rate for encoding, so the framerate is not reduced on static content.

**Comment by linckosz** [comment]
I am using Sunshine for a desktop use, having low fps (~10fps) when I move around the mouse generate a latency that makes the mouse pointing quite difficult and slow down an office work.
Being able to **set a minimum FPS manually** (like 60fps for office and 90fps for game) would be very nice.

**Comment by lastmetroid** [comment]
Same issue here, especially with AV1, stream starting to lag with no motion and get high latency of 60-80ms, see issue here:

https://github.com/moonlight-stream/moonlight-qt/issues/1296

Is there no way in Sunshine to force a constant FPS like 60 or 120? It seems to throttle down the FPS on the fly with no motion which causes issues on the client.
